mlrun/common/runtimes/constants.py

```diff
@@ -193,8 +195,9 @@ def not_allowed_for_deletion_states():
 
 
 class RunLabels(enum.Enum):
-    owner = "owner"
-    v3io_user = "v3io_user"
+    owner = MlrunInternalLabels.owner
+    # TODO: remove this in 1.9.0
```

Comment:
this todo should be on the entire class and explain that we need to use only MlrunInternalLabels by then